fitbenchmarking/fitbenchmarking
# Add DFO-LS

#### Description of Work

Fixes #419 by adding dfo-ls
Changes name of the controller from DFOGN to DFO

#### Testing Instructions

1. Run with the dfogn/dfols options
2. Check all works ok

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
